{
	"schema_version": "1.6.0",
	"id": "DSA-4892-1",
	"modified": "2024-01-09T20:29:11.329989Z",
	"published": "2021-04-18T00:00:00Z",
	"related": [
		"CVE-2021-23980"
	],
	"summary": "python-bleach - security update",
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:10",
				"name": "python-bleach",
				"purl": "pkg:deb/debian/python-bleach?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.1.2-0+deb10u2"
						}
					]
				}
			],
			"versions": [
				"3.1.0-1",
				"3.1.0-2",
				"3.1.1-0+deb10u1",
				"3.1.1-1",
				"3.1.2-0+deb10u1"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/debian-osv/dsa-osv/DSA-4892-1.json"
			}
		}
	]
}
